### PR TITLE
GH#15186 - Can specify an MTU manually

### PR DESCRIPTION
**Pull request by jboxman**
- https://github.com/openshift/openshift-docs/issues/15186

### CODE REVIEW

**Comment by jboxman** [comment]
@danwinship, this adds a missing Ansible variable for OpenShift SDN `mtu` size in OCP 3.11. What do you think? Thanks!

File: install/configuring_inventory_file.adoc [hunk 1]
```diff
@@ -400,6 +400,9 @@ the default SDN. If enabling *flannel*, disable the default SDN with the
 |`openshift_sdn_vxlan_port`
 |This variable sets the `vxlan port` number for `cluster network`. Defaults to `4789`. See xref:../install_config/configuring_sdn.html#config-changing-vxlan-port-for-cluster-network[Changing the VXLAN PORT for the cluster network] for more information.
 
+|`openshift_node_sdn_mtu`
+|This variable specifies the MTU size to use for OpenShift SDN. The value must be `50` bytes less than the MTU of the primary network interface of the node. For example, if the primary network interface has an MTU of `1500`, this value will be `1450`. The MTU value is discovered automatically but can be overwritten if necessary.
```
Comment (danwinship) [inline install/configuring_inventory_file.adoc:404]: The "discovered automatically" part is a 4.x feature. In 3.x it defaults to 1450 unconditionally, so if your nodes don't have MTU 1500, you definitely need to override this.

This is mentioned in a few other other places in the docs already, but they all talk about editing the node config directly, not using ansible, so maybe there should be some updating/xreffing. eg, `install_config/configuring_sdn.adoc`, `day_two_guide/topics/verifying_mtu.adoc`, `scaling_performance/network_optimization.adoc`, `admin_guide/sdn_troubleshooting.adoc`

Comment (jboxman) [reply install/configuring_inventory_file.adoc:404]: @danwinship, ah, cool, glad I asked!

Comment (jboxman) [reply install/configuring_inventory_file.adoc]: @danwinship, I've omitted the automatic discovery sentence. OCP 3.x is old enough that I'm hesitant to rework any of the other content that discusses modifying the MTU on the node to incorporate this.

**Comment by danwinship** [comment]
The text that's there now is correct then. The only thing you might want to add is that it's 1450 by default so you wouldn't actually need to specify it in that case. (Maybe say that, eg, when deploying on AWS where nodes have MTU 9000, you should set this to 8950.)

**Comment by danwinship** [comment]
actually maybe don't say that because we might already change it automatically in the AWS case? But anyway, 1450 is the default normally. That may be worth mentioning.

**Review by codyhoag — APPROVED** [review]
LGTM!

**Comment by jboxman** [comment]
@codyhoag thanks!

**Pull request merged**